tests/kernel/usleep/src/main.c

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
can this test be added to existing sleep test? Adding a test per API call does not scale.

Reply:
Ah, indeed it can.  It was originally separated out because `prj.conf` had a manual setting for `CONFIG_SYS_CLOCK_TICKS_PER_SEC`, but that's not necessary now.  Moved to the existing sleep test.